Docs/Entities.md

```diff
@@ -0,0 +1,31 @@
+# Entities
+
+### User
+- ID ```number```
+- First Name ```string(25)```
+- Last Name ```string(25)```
+- Email ```string(100)```
+- Password ```string(100)```
+- Country ```string```
```

Comment:
Why we need to know the user's country?

Reply:
It's for a future planning to get analysis from their region only (Jobs analysis).